diff --git a/src/app/api/auth/login/route.ts b/src/app/api/auth/login/route.ts
index d4ab77f..a5d33cb 100644
--- a/src/app/api/auth/login/route.ts
+++ b/src/app/api/auth/login/route.ts
@@ -7,25 +7,6 @@ export const dynamic = "force-dynamic";
 export const revalidate = 0;
 export const fetchCache = "force-no-store";
 
-// Testing the route
-export async function GET() {
-  const supabase = await createClient();
-  const { error } = await supabase.from("posts").select("*");
-
-  if (error) {
-    return NextResponse.json({ error: error.message }, { status: 400 });
-  }
-
-  const res = NextResponse.json(
-    { message: "Auth login Route Accessed Successfully!" },
-    { status: 200 }
-  );
-  res.headers.set("Cache-Control", "no-store, no-cache, must-revalidate, proxy-revalidate");
-  res.headers.set("Pragma", "no-cache");
-  res.headers.set("Expires", "0");
-  return res;
-}
-
 export async function POST(req: NextRequest) {
   const { email, password } = await req.json();
   const supabase = await createClient();
